src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -193,7 +193,7 @@ function SendTokenTransaction() {
         <ButtonWrapper>
           <Button onPress={handleDismissal} title={successButtonText} />
         </ButtonWrapper>
-        <TouchableOpacity onPress={handleDismissal}>
+        <TouchableOpacity onPress={handleDismissal} style={styles.viewBlockchainButton}>
```

Comment:
Button should not be placed inside TouchableOpacity. In order to handle both dismissal & viewOnBlockchain pls put dismiss call into viewOnBlockchain function.